assets/vendor-fbfac994e6d2a78bf06e7e9f994909cd.js

```diff
@@ -65286,8 +65286,7 @@ module.exports=window.L=require("leaflet/dist/leaflet-src");
 
 ;/* FileSaver.js
  * A saveAs() FileSaver implementation.
- * 1.3.4
- * 2018-01-12 13:14:0
+ * 1.2.2
```

Comment:
Why downgrade FileSaver.js?

Reply:
They totally change the file organization and recent version are not compatible with station studio. It's probably not a big change.